packages/ui/src/components/Generic/SessionCountIcon/SessionCountIcon.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { defaultTheme } from "@repo/theme"
+import type { Meta, StoryObj } from "@storybook/react"
+import { SessionCountIcon } from "./SessionCountIcon"
+
+const fontSizes = Object.keys(defaultTheme.fontSizes ?? {})
+
```

Comment:
You can just import the tokens.
```suggestion
import { tokens } from "@repo/theme"
import type { Meta, StoryObj } from "@storybook/react"
import { SessionCountIcon } from "./SessionCountIcon"

const fontSizes = Object.keys(tokens.fontSizes)

```